src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -398,6 +401,10 @@ export const ArtistAuctionResultsRefetchContainer = createRefetchContainer(
         slug
         internalID
         name
+        meta {
```

Comment:
So something like `meta(page: AUCTION_RESULTS) { ... }` would be preferable and more clear.